src/sentry/audit_log/register.py

```diff
@@ -0,0 +1,626 @@
+    role = audit_log_entry.data.get("role") or "N/A"
+
+    if "team_slugs" in audit_log_entry.data:
+        teams = ", ".join(str(x) for x in audit_log_entry.data.get("team_slugs", []))
+    else:
+        teams = "N/A"
+
+    return f"edited member {member} (role: {role}, teams: {teams})"
+
+
+def render_member_remove(audit_log_entry: AuditLogEntry):
+    if audit_log_entry.target_user == audit_log_entry.actor:
+        return "left the organization"
+
+    member = audit_log_entry.data.get("email") or audit_log_entry.target_user.get_display_name()
+
+    return f"removed member {member}"
+
+
+def render_member_join_team(audit_log_entry: AuditLogEntry):
+    if audit_log_entry.target_user == audit_log_entry.actor:
+        return "joined team {team_slug}".format(**audit_log_entry.data)
+
+    user_display_name = (
+        audit_log_entry.data.get("email") or audit_log_entry.target_user.get_display_name()
+    )
+    return "added {} to team {team_slug}".format(user_display_name, **audit_log_entry.data)
+
+
+def render_member_leave_team(audit_log_entry: AuditLogEntry):
+    if audit_log_entry.target_user == audit_log_entry.actor:
+        return "left team {team_slug}".format(**audit_log_entry.data)
+
+    user_display_name = (
+        audit_log_entry.data.get("email") or audit_log_entry.target_user.get_display_name()
+    )
+    return "removed {} from team {team_slug}".format(user_display_name, **audit_log_entry.data)
+
+
+def render_member_pending(audit_log_entry: AuditLogEntry):
+    user_display_name = (
+        audit_log_entry.data.get("email") or audit_log_entry.target_user.get_display_name()
+    )
+    return f"required member {user_display_name} to setup 2FA"
+
+
+def render_org_edit(audit_log_entry: AuditLogEntry):
+    items_string = ", ".join(f"{k} {v}" for k, v in audit_log_entry.data.items())
+    return "edited the organization setting: " + items_string
+
+
+def render_project_edit(audit_log_entry: AuditLogEntry):
+    if "old_slug" in audit_log_entry.data:
+        return (
+            "renamed project slug from "
+            + audit_log_entry.data["old_slug"]
+            + " to "
+            + audit_log_entry.data["new_slug"]
+        )
+    items_string = " ".join(f"in {key} to {value}" for (key, value) in audit_log_entry.data.items())
+    return "edited project settings " + items_string
+
+
+def render_project_enable(audit_log_entry: AuditLogEntry):
+    # Most logs will just be name of the filter, but legacy browser changes can be bool, str or sets
+    filter_name = audit_log_entry.data["state"]
+    if filter_name in ("0", "1") or isinstance(filter_name, set) or isinstance(filter_name, bool):
+        message = "enabled project filter legacy-browsers"
+        if isinstance(filter_name, set):
+            message += ": {}".format(", ".join(filter_name))
+        return message
+    return f"enabled project filter {filter_name}"
+
+
+def render_project_disable(audit_log_entry: AuditLogEntry):
```

Comment:
Nit: The logic for project enable and disable is similar. Could this be broken out into a separate function?